### DISCUSSION TITLE
Documentation: Add more complex example for the tree element

### DISCUSSION THREAD

**Discussion by gavinpotter** [zauberzeug/nicegui#1660] [Ideas / Feature Requests]
The documentation correctly states that the nodes must be presented in a heirarchy.  Its not immediately clear what this means, I think a more complex example would help.

Something like:
[{'id': 'root', 'children': [{'id': 'child 1', 'children': [{'id': 'child11'}]}, {'id': 'child 2'}]}]

Its quite difficult to produce this programatically.  I had to resort to ChatGPT for help!

**Comment by rodja** [comment]
Would you like to create a pull request?

**Comment by natankeddem** [comment]
You might want to utilize the ui.json_editor to create complex data graphically if that is easier for you.
